web-docs/docs/02-CHANGELOG.mdx

```diff
@@ -5,6 +5,12 @@ sidebar_label: Change Log
 toc_max_heading_level: 2
 ---
 
+## Version 1.15.11
+### Features
+- Added support for MQTT will
+  - Commonly referred to as "Last Will and Testament" (LWT)
+  - Configurable topic, payload, and retain setting
```

Comment:
```suggestion
- Added support for MQTT will (LWT/Last Will and Testament)
  - Configurable topic, payload, and retain setting
```
Optional, but I think this could be cleaned up and simplified.

Reply:
combined elements of both